cves/kernel/CVE-2014-3183.yml

```diff
@@ -115,6 +130,7 @@ upvotes_instructions: |
   interesting they think this vulnerability is, and you'll add that to the
   upvotes score on your branch.
 upvotes:
```

Comment:
I would give this vulnerability a seven out of ten as it is both information disclosure and elevation of privileges or possibly crash the system.

Reply:
Added